prepare-for-migration.py

```diff
@@ -0,0 +1,98 @@
+import os
```

Comment:
**chore (blocking):** it looks like there are specific implementation details in this script. let's remove this from the current PR and discuss.